pkg/core/config/config.go

```diff
@@ -1,5 +1,10 @@
 package config
 
+const (
+	// FuseMlNamespace is the kubernetes namespace where FuseML workfloads are created
+	FuseMlNamespace = "fuseml-workloads"
```

Comment:
Could it be `FuseMLNamespace` instead? Or would it be against some go style?

Reply:
done!